package.json

```diff
@@ -131,12 +131,20 @@
       }
     },
     "rules": {
-      "@typescript-eslint/no-unused-vars": [
+      "import/no-anonymous-default-export": [
         1,
         {
-          "vars": "local"
+          "allowObject": true
         }
       ],
+      "react/jsx-pascal-case": [
+        2,
+        {
+          "allowNamespace": true
+        }
+      ],
```

Comment:
I had to upgrade eslint and apply this rule to allow `Primitive.button` (so lowercase `button`).